emag-tech-labs/annotation-cache-bundle/5.0/config/packages/emag_annotation_cache.yaml

```diff
@@ -0,0 +1,3 @@
+annotation_cache:
+    provider: {}
+    ignore_namespaces: []
```

Comment:
We can remove this. Let's add this to the docs instead. 

Alternatively, add `App\Entity` here. They are already cached by doctrine.

Reply:
Added 'App\Entity'